### ISSUE TITLE
find in files fails to find partial word match

### ISSUE THREAD

**Issue by ahcyip** [rstudio/rstudio#5217]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.2.1335
    OS Version      : Windows 10 Pro
    R Version       : 3.5.1

### Describe the problem in detail
Find in Files fails to find a partial word match when I try to find the search term e.g. VattAll (and others). I get the message "no results found". But If I Find in Files "VattAll.xlsx")), I get a result.

### Describe the behavior you expected
I expected to find the instance of "VattAll.xlsx")) in the code when searching for VattAll. In fact, I do find it (and other matches) when using my other computer on the same code/Rstudio project, with same versions of Rstudio, R, and Windows 10 Home instead of Pro.

It seems Rstudio is not finding matches unless I include all punctuation and characters between spaces.

**Comment by ronblum** [comment]
@ahcyip Thank you for reporting this issue. I'm unable to reproduce it, though, both on Windows 10 Pro and MacOS 10.15 (Catalina). Could you provide a code sample for which this problem occurs so that we can try further to reproduce the problem?

**Comment by ahcyip** [comment]
Unfortunately I encounter the problem on one computer and not the other. I've been able to further isolate the problem (on the one computer): search string works if one end has a space or if is made of only a-z characters, but fails if neither end has a space. 

On one computer,
e.g. 
Find All Files for 
esearchfolderpat finds all instances of researchfolderpath, so no problems here

in the code: read_excel(file.path(researchFolderPath, "data", "VattAll.xlsx"))
VattAll.xlsx")) finds this
VattAll.xlsx") finds this
VattAll.xlsx" finds this
"Vatt finds this

VattAll.xlsx fails to find this ("No results found")
VattAll fails to find this ("No results found")

Again, I only encounter this issue on one computer (where I just updated RStudio 1.2.1335 Build 1379 (f1ac3452)) and not the other (also 1.2.1335 Build 1379 (f1ac3452)) (same rstudio project / github repository).

Language issue? regex issue?
